src/System.Management.Automation/engine/remoting/client/remoterunspace.cs

```diff
@@ -2610,6 +2793,42 @@ private void SetIsActive(int breakpointCount)
             }
         }
 
+        private T InvokeVirtualBreakpointFunction<T>(string functionName, Dictionary<string, object> parameters)
```

Comment:
Name should be 'InvokeRemoteBreakpointFunction'.

Reply:
Fixed in next commit.